static/build.yml

```diff
@@ -5,9 +5,9 @@ dependencies:
     - ..  # d2l
     - git+https://github.com/d2l-ai/d2l-book
     - mxnet-cu102==1.7.0
-    - torch==1.11.0+cu102
+    - torch==1.12.0+cu102
     - -f https://download.pytorch.org/whl/torch_stable.html
-    - torchvision==0.12.0+cu102
+    - torchvision==0.13.0+cu102
     - -f https://download.pytorch.org/whl/torch_stable.html
-    - tensorflow==2.8.0
-    - tensorflow-probability==0.16.0
+    - tensorflow==2.9.1
+    - tensorflow-probability==0.17.0
```

Comment:
Since we are updating libraries here, I'd suggest let's rerun the CI from scratch here for the whole project before merging.

Reply:
OK. Makes sense. But it failed again in the rebuild at another notebook creation and it looks like I do not have permissions to activate d2l-bot actions. @AnirudhDagar, could you, kindly, either rebuild or give me more power to rule d2l-bot :D ?